src/autocomplete.css

```diff
@@ -29,6 +29,14 @@
   outline: 3px solid #ffbf47;
 }
 
+.autocomplete__input.autocomplete__input--show-all-values {
```

Comment:
I personally never rely on the cascade or specificity when I use BEM.

I would add another modifier class, like `--default`, which adds the implicit padding and make the code never use both `--default` or `--show-all-values` simultaneously.

So in BEM, I never do this:

```css
.button {
  background-color: blue;
  color: white;
}

.button--primary {
  background-color: green;
}
```

Rather, I do this:

```css
.button {
  color: white;
}

.button--default {
  background-color: blue;
}

.button--primary {
  background-color: green;
}
```

`.button` on its own makes no sense in this example, and it's fine when you have a component system because you can encapsulate this contract neatly away. (rather than relying on users adding the `class="button button--default"` to a million places in their codebase, which is brittle)